src/main/java/de/geolykt/enchantments_plus/Config.java

```diff
@@ -33,43 +42,154 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import de.geolykt.enchantments_plus.compatibility.enchantmentgetters.AdvancedLoreGetter;
-import de.geolykt.enchantments_plus.compatibility.enchantmentgetters.BasicLoreGetter;
-import de.geolykt.enchantments_plus.compatibility.enchantmentgetters.LeightweightPDCGetter;
-import de.geolykt.enchantments_plus.compatibility.enchantmentgetters.PersistentDataGetter;
-import de.geolykt.enchantments_plus.enchantments.*;
-import de.geolykt.enchantments_plus.enums.BaseEnchantments;
-import de.geolykt.enchantments_plus.evt.WatcherEnchant;
-import de.geolykt.enchantments_plus.util.AreaOfEffectable;
-import de.geolykt.enchantments_plus.util.Tool;
-
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.reflect.InvocationTargetException;
 import java.util.*;
 
+enum ConfigKeys {
```

Comment:
You should put the public class first, then the non-public ones by convention (just for future's sake and so you know why I'm reverting some changes of yours)